debezium-ddl-parser/src/test/resources/mysql/examples/ddl_alter.sql

```diff
@@ -2,10 +2,12 @@
 -- Alter Table
 alter table ship_class add column ship_spec varchar(150) first, add somecol int after start_build, algorithm=instant;
 alter table t3 add column (c2 decimal(10, 2) comment 'comment`' null, c3 enum('abc', 'cba', 'aaa')), add index t3_i1 using btree (c2) comment 'some index';
+alter table t3 add column (c4 decimal(10, 2) comment 'comment`' null, add index t3_i2 using btree (c4) comment 'some index';
```

Comment:
A closing brace is missing here after the column definition. I'll add it.

Reply:
Hum, the `ifExists` clause hasn't been added to `alterSpecification`. Seems like the core of this change is missing actually.